cmd/kubeadm/app/util/config/common.go

```diff
@@ -44,6 +44,14 @@ import (
 	kubeadmutil "k8s.io/kubernetes/cmd/kubeadm/app/util"
 )
 
+// LoadOrDefaultConfigurationOptions holds the common LoadOrDefaultConfiguration options.
+type LoadOrDefaultConfigurationOptions struct {
+	// Allow migration to experimental, unreleased APIs
```

Comment:
The comment should start with the field name, it's a good practice, such as:
```suggestion
	// AllowExperimental indicates whether the experimental / work in progress APIs can be used.
```